permanent_upload/validation.py

```diff
@@ -2,4 +2,19 @@
 
 
 def validate_supported_types(results, data_file="data/supported_file_types.csv"):
-    return True
+    validation_dataset = {}
+    with open(data_file) as csvfile:
+        validation_reader = csv.reader(csvfile)
+        for row in validation_reader:
+            validation_dataset[row[0]] = row[1:]
+    for result in results:
+        extension = result[0].split(".")[1]
```

Comment:
You want the last period-separated segment, not the second:

```suggestion
        extension = result[0].split(".")[-1]
```